.github/PULL_REQUEST_TEMPLATE/documentation_template.md

```diff
@@ -0,0 +1,69 @@
+# Documentatioon 
+
+**Title:** 
+
+## Issue reference
+
+**Issue:** *issue number*
+
+## Category
+
+Which category does this documentation add/updated/remove?
+- [ ] code documentation
+- [ ] wiki
+- [ ] README
+- [ ] installation guide
+    - [ ] OS specifics
+    - [ ] dependency
+- [ ] tutorials 
+    - [ ] plotting
+    - [ ] io
+    - [ ] data products 
+- [ ] data product information 
+- [ ] examples
+- [ ] citing/licensing
+- [ ] trouble shooting
+    - [ ] installation
+    - [ ] tutorials
+- [ ] history
+- [ ] guidelines 
+    - [ ] developer
+    - [ ] communication
+    - [ ] release
```

Comment:
These types of guidelines are not part of the RST documentation (they live on the wiki instead). So maybe this section will never be used?
```suggestion
```

Reply:
I agree, maybe this is something to add onto as an issue as that might be a better github route for input on these types of documents.

Reply:
Okay, can people not edit the wiki?

Reply:
@mts299 Can we remove this section, since it probably won't ever be used in a PR?